core/include/detray/core/detail/container_buffers.hpp

```diff
@@ -232,14 +258,20 @@ dvector_view<T> get_data(dvector_buffer<T>& buff) {
 /// @c mutli_view to be ultimately passed on to the class constructor.
 ///
 /// @note This does not pick up the vecmem types.
-template <class... Ts>
-auto get_data(dmulti_buffer<Ts...>& multi_buff) {
+template <class... Ts, std::size_t... I>
```

Comment:
THe grid views have the same type, so they need to be fetched by index from the tuple